### ISSUE TITLE
Have cloze-prefix and cloze-suffix include selected subtitles in subtitle list

### ISSUE THREAD

**Issue by ElaxDev** [ripose-jp/Memento#173]
I have the following configuration for my Anki cards in the sentence field: `{cloze-prefix}<b>{cloze-body}</b>{cloze-suffix}` so I can have the term in the sentence highlighted, but if I select let's say the subtitle line before the one where I mined the word it just puts the current subtitle line in that field, and I think the best way to fix that could be for cloze-prefix to include the text in the subtitles selected that are before cloze-body, and for cloze-suffix to include the text that is in the subtitles selected that are after cloze-body

**Comment by ripose-jp** [comment]
If I'm understanding correctly, you want a version of cloze that takes the cloze from the subtitle list. Essentially `{context-prefix}` and `{context-suffix}` markers.

When you write "subtitle line" I get a little confused about what you mean since a single subtitle can be split across newlines. I just want to make sure I'm not misunderstanding this as a feature request as opposed to a bug report.

**Comment by ElaxDev** [comment]
Yes, that's exactly what I want, it is a feature request, sorry for not being more clear.
